### ISSUE TITLE
Packaging request: Meshroom

### ISSUE THREAD

**Issue by aidalgol** [NixOS/nixpkgs#191716]
## Project description
Meshroom is a tool for reconstructing a 3D object from a sequence of still images or video.

## Metadata
* homepage URL: https://alicevision.org/#meshroom
* source URL: https://github.com/alicevision/meshroom
* license: mpl20
* platforms: linux

## Additional Info
I have [a branch](https://github.com/aidalgol/nixpkgs/tree/meshroom) that adds all of Meshroom's dependencies, but getting Meshroom itself packaged is blocked on #191313.  This can be seen by checking out my `nixpkgs` branch, and the `2021.1.0` branch of the `meshroom` repository, and then running `python meshroom/ui` inside a `nix-shell`, with the below `shell.nix` file

<details>
<summary>meshroom invocation output</summary>

```console
$ NIXPKGS_ALLOW_UNFREE=1 NIX_PATH="$HOME/src" nix-shell --run "PYTHONPATH=$PWD python meshroom/ui"
[2022-09-18 10:56:32,155][WARNING] == The following "aliceVision" plugins could not be loaded ==
  * SketchfabUpload: No module named 'requests'

[2022-09-18 10:56:32,157][WARNING] == The following "submitters" plugins could not be loaded ==
  * simpleFarmSubmitter: No module named 'simpleFarm'

Traceback (most recent call last):
  File "/nix/store/c1vb2z3c64i0sd92iz7fv0lb720qcvhb-python3-3.10.6/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/nix/store/c1vb2z3c64i0sd92iz7fv0lb720qcvhb-python3-3.10.6/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/aidan/src/meshroom/meshroom/ui/__main__.py", line 10, in <module>
    from meshroom.ui.app import MeshroomApp
  File "/home/aidan/src/meshroom/meshroom/ui/app.py", line 17, in <module>
    from meshroom.ui.components.scene3D import Scene3DHelper, Transformations3DHelper
  File "/nix/store/mx7gf4v9cai3srkmllg4bdyh14rcv62h-shiboken2-5.15.5/lib/python3.10/site-packages/shiboken2/files.dir/shibokensupport/feature.py", line 139, in _import
    return original_import(name, *args, **kwargs)
  File "/home/aidan/src/meshroom/meshroom/ui/components/scene3D.py", line 4, in <module>
    from PySide2.Qt3DCore import Qt3DCore
  File "/nix/store/mx7gf4v9cai3srkmllg4bdyh14rcv62h-shiboken2-5.15.5/lib/python3.10/site-packages/shiboken2/files.dir/shibokensupport/feature.py", line 139, in _import
    return original_import(name, *args, **kwargs)
ModuleNotFoundError: No module named 'PySide2.Qt3DCore'
```
</details>

<details>
<summary>meshroom shell.nix</summary>

```nix
{ pkgs ? import <nixpkgs> {} }:

pkgs.mkShell {
  packages = with pkgs; [
    alicevision
    (python3.withPackages (pypkgs: with pypkgs; [
      pyside2
      psutil
    ]))
  ];
}
```
</details>

**Comment by urandom2** [comment]
this is a duplicate of #94127 and #171324
